client/src/components/solution-display-widget/style.css

```diff
@@ -0,0 +1,4 @@
+.w-100,
+.w-100 > .dropdown-menu {
+  width: 100%;
+}
```

Comment:
```suggestion
```

Reply:
Hey I looked into your proposed solution `col-xs-12`, it didn't worked for `MenuItem`
`DropdownButton` and `MenuItem` both should have `width` of `100%`. Otherwise only `View` button will have full width and not `MenuItem`

using my custom css `w-100`
![image](https://user-images.githubusercontent.com/91016903/202535255-b4918dcb-13e3-4160-a867-795eb85ef131.png)


using `col-xs-12`
![image](https://user-images.githubusercontent.com/91016903/202535141-4584794d-dbb5-4981-ae76-cbc706862d10.png)

Which one you would prefer ?

Reply:
Thank you for checking it out, if it doesn't look right. Let me see if there is a class in the codebase there with the changes, that make our life easier to update all of them later.

Reply:
Sure, `.dropdown-menu` is the class of `ul` element which `MenuItem` adds dom. I'm using that class  to update its width to 100%

Reply:
Oh, I mean using the styles in `time-line.css`, as this component is doing the same thing and almost have the same style, we can use the same file

Reply:
from `timeline.css`
```css
.timeline-row .solutions-dropdown .dropdown {
  width: 100%;
}
```
It didn't worked as `.dropdown` don't have any parent element with class `solutions-dropdown` whose parent class is `timeline-row`

Reply:
True, adding `<div className='.solutions-dropdown'>` as parent gives width `100%`.

![image](https://user-images.githubusercontent.com/88248797/202775891-ebc501bc-0490-4107-a5a1-f12c724f1704.png)

and it will look the same as functions below it 

https://github.com/freeCodeCamp/freeCodeCamp/blob/6ad3a78db1fd6bf37a56db700fc33e3fb9cb424a/client/src/components/solution-display-widget/index.tsx#L90-L105

Reply:
I have been checking the wrong certification the whole time?

Reply:
Nope, adding parent element with class name as `.solutions-dropdown` didn't worked either. Check quality-assurance-v7 certification.